tools/istio-iptables/pkg/capture/run_linux.go

```diff
@@ -0,0 +1,841 @@
+// Copyright Istio Authors
```

Comment:
Do we need to mvoe all of this to run_linux.go?  Could only `ConfigureRoutes` be moved to a new file?

Reply:
Pass #2 with just moving the `configureTProxyRoutes` to a _linux file and stubbing it out for all non-linux builds.